waiter/src/waiter/metrics.clj

```diff
@@ -473,3 +474,10 @@
    :stream-read-body (service-timer service-id "stream-read-body")
    :stream-request-rate (service-meter service-id "stream-request-rate")
    :throughput-meter (service-meter service-id "stream-throughput")})
+
+(defn report-duration
+  "Report elapsed duration on the given timer metric. Returns nil."
```

Comment:
Please clarify in the docstring that the granularity is limited to `milliseconds` to warn future users who might want a finer granularity.

Reply:
Done.